wp-seo-main.php

```diff
@@ -597,3 +597,19 @@ function yoast_wpseo_self_deactivate() {
 		}
 	}
 }
+
+/**
+ * Returns the notice for the permalink settings page
+ */
+function yoast_wpseo_permalink_warning() {
+	global $pagenow;
+
+	if ( $pagenow === 'options-permalink.php' ) {
+		$warning = esc_html__( 'WARNING:', 'wordpress-seo' );
+		$message = __( 'Changing your permalinks settings can seriously impact your search engine visibility. It should almost <i>never</i> be done on a live website.', 'wordpress-seo' );
```

Comment:
Can you make sure there are no `<i>` html elements in the translatable string? I would suggest to use `sprintf` for this. For example:
``` 
/* translators: %1$s and %2$s expand to <i> items to emphasize the word in the middle. */
$message = esc_html__('Changing your permalinks settings can seriously impact your search engine visibility. It should almost %1$s never %2$s be done on a live website.', 'wordpress-seo');

echo sprintf($message, '<i>', '</i>');
```
Now you can also use `esc_html__` here.